node/http/src/types.rs

```diff
@@ -14,6 +14,13 @@ use primitives::types::{
     AccountId, AuthorityStake, Balance, BlockIndex, MerkleHash, Nonce, ShardId,
 };
 
+pub enum RPCError {
+    BadRequest(String),
```

Comment:
do we want some sort of generic "server error" in addition to these?

Reply:
Prefer adding things when we actually need them.